### PR TITLE
Feature/derp 240 manage layout row component

### PR DESCRIPTION
**Pull request by Fullstack-Lincoln**

### CODE REVIEW

**Comment by philwolstenholme** [comment]
Linc and I spoke about this and we'll wait a bit longer to merge it in as Linc is still working on the backend display

File: modules/gla_core_layout/gla_core_layout.module [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+/**
+ * Implements hook_field_widget_complete_form_alter().
+ */
+function gla_core_layout_field_widget_complete_form_alter(&$field_widget_complete_form, FormStateInterface $form_state, $context) {
+  if ($field_widget_complete_form['widget']['#field_name'] == 'field_p_ti_components') {
+    if (in_array('field_p_lr_columns', $field_widget_complete_form['#parents'], TRUE)) {
+      // Remove Layout row option from Tab item paragraphs add more button to
+      // avoid issues with nested layout rows.
+      unset($field_widget_complete_form['widget']['add_more']['add_more_button_layout_row']);
+      unset($field_widget_complete_form['widget']['add_more']['operations']['#links']['add_more_button_layout_row']);
```
Comment (VincenzoGambino) [inline modules/gla_core_layout/gla_core_layout.module:40]: Could this be achieved using ['#access'] = FALSE; ?

Comment (Fullstack-Lincoln) [reply modules/gla_core_layout/gla_core_layout.module:40]: Ermmm I don't know 100% yet, this code won't be in use yet until we build the tabs component....

Comment (philwolstenholme) [reply modules/gla_core_layout/gla_core_layout.module:40]: I don't think we're are going to have a Tab component @Fullstack-Lincoln so I would delete this function, we shouldn't copy across code that we don't know for sure is going to be used in the future.

File: modules/gla_core_layout/src/Plugin/Layout/MultiWidthSpacingLayout.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      ];
+    }
+    return $configuration;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    // Spacing, vertical and horizontal alignment use Bootstrap classes to save
```
Comment (philwolstenholme) [inline modules/gla_core_layout/src/Plugin/Layout/MultiWidthSpacingLayout.php:37]: Swap these Bootstrap mentions for Tailwind I think @Fullstack-Lincoln?

**Review by philwolstenholme — CHANGES_REQUESTED** [review]

File: modules/gla_core_layout/src/Plugin/Layout/MultiWidthSpacingLayout.php (hunk 2 above)
Comment (Fullstack-Lincoln) [reply modules/gla_core_layout/src/Plugin/Layout/MultiWidthSpacingLayout.php]: Done

**Review by philwolstenholme — APPROVED** [review]

**Pull request merged**